### PR TITLE
[TIMOB-23398] Analytics: SDK version format on Windows phone should match other platforms

### PR DESCRIPTION
**Pull request by sgtcoolguy**
https://jira.appcelerator.org/browse/TIMOB-23398

We need to prefix sdkver value with "ti." and trim the version down to the first three segments (drop the timestamp qualifier).

### CODE REVIEW

**Comment by infosia** [comment]
build win 10

**Comment by infosia** [comment]
build win 8.1

**Comment by infosia** [comment]
Jenkins says `L` is not found, I think this means `Ti.Analytics._start();` throws exception during `Analytics::loadJS` in this case. From Jenkins console:

```javascript
22:09:39 [ERROR]: Failed to initialize Titanium: SyntaxError: Unexpected token ';'. Expected '}' to end a object literal.
```

File: Source/TitaniumKit/src/analytics.js [hunk 1]
```diff
@@ -93,7 +103,7 @@ Analytics.prototype.createEnrollEvent = function createAppEnrollEvent() {
 		nettype: Ti.Network.networkTypeName,
 		app_version: Ti.App.version,
 		osver: Ti.Platform.version,
-		sdkver: Ti.version
+		sdkver: "ti." + trimVersionQualifier(Ti.version);
```
Comment (garymathews) [inline Source/TitaniumKit/src/analytics.js]: Typo of ``;`` character

**Comment by garymathews** [comment]
CR: Fix ``;`` typo

FR: After fixing typo
###### BEFORE
```Javascript
{
 ...
 "sdkver": "5.4.0.v20160502140437"
}
```

###### AFTER
```Javascript
{
 ...
 "sdkver": "ti.5.4.0"
}
```

File: Source/TitaniumKit/src/analytics.js [hunk 2]
```diff
@@ -116,7 +126,7 @@ Analytics.prototype.createForegroundEvent = function foregroundEvent() {
...
```
Comment (garymathews) [inline Source/TitaniumKit/src/analytics.js]: Typo of ``;`` character

**Comment by garymathews** [comment]
build win 10

**Comment by garymathews** [comment]
build win 10

**Comment by infosia** [comment]
Looks good, just waiting for update with the base branch.

**Pull request merged**